### PR TITLE
fix: syntax problems from branch 2021.x

### PR DESCRIPTION
**Pull request by demoManito**
### Describe what this PR does / why we need it
fix syntax problems

### CODE REVIEW

File: spring-cloud-alibaba-starters/spring-cloud-alibaba-sentinel-gateway/README.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ Sentinel has full rule config features. see [Dynamic-Rule-Configuration](https:/
 Implements `SentinelFallbackProvider` to define your own Fallback Provider when Sentinel Block Exception throwing for different rout. the default 
 Fallback Provider is `DefaultBlockFallbackProvider`. 
 
-By default Fallback route is `ServiveId + URI PATH`, example `/book/coke`, first `book` is serviceId, `/uri` is URI PATH, so both  
+By default, Fallback route is `ServiveId + URI PATH`, example `/book/coke`, first `book` is serviceId, `/uri` is URI PATH, so both
```
Comment (steverao) [inline spring-cloud-alibaba-starters/spring-cloud-alibaba-sentinel-gateway/README.md]: Fallback should be fallback.

**Comment by steverao** [comment]
Please create a same PR to branch of 2021.x.

**Comment by demoManito** [comment]
@steverao please review again

**Comment by steverao** [comment]
> @steverao please review again

we maintain branch of 2.2.x and 2021.x, you should create a same PR to another.

**Comment by demoManito** [comment]
#2768 like this?

**Pull request merged**